packages/@adobe/spectrum-css-temp/components/inputgroup/skin.css

```diff
@@ -258,8 +270,10 @@ governing permissions and limitations under the License.
     }
 
     &.is-disabled:not(.spectrum-InputGroup--invalid):not(.spectrum-InputGroup--quiet) .spectrum-FieldButton {
-      border-color: GrayText;
       border-width: 1px;
+      &:before {
```

Comment:
I think we need to delete the border-width on the line above this as well
See https://reactspectrum.blob.core.windows.net/reactspectrum/c66b7154aec76fef87da719f58afbc190fea70a3/storybook/index.html?path=/story/combobox--default&args=isDisabled:true&providerSwitcher-express=false:
![Screenshot 2023-05-03 at 11 46 50 AM](https://user-images.githubusercontent.com/698229/236014351-3a72cff5-e797-4f7c-a760-c17ca5e2c3ab.png)


VS if we turn off that property
![Screenshot 2023-05-03 at 11 47 10 AM](https://user-images.githubusercontent.com/698229/236014476-2fc8eb18-4eaa-4ae0-8466-c70a12114bba.png)